plugin/StrikeLightningConnectionStringHandler.cs

```diff
@@ -94,51 +101,12 @@ public StrikeLightningConnectionStringHandler(IServiceProvider serviceProvider,
 		if (serverUrl != null)
 			client.ServerUrl = serverUrl;
 
+		// initialize lightning client that listens on top of StrikeClient
 		var logger = _loggerFactory.CreateLogger<StrikeLightningClient>();
-
-		var connectionHash = ComputeHash(connectionString);
-		var accountFiatCurrency = GetAccountFiatCurrency(connectionHash, client, ref error);
-		if (accountFiatCurrency == null)
-			return null;
-
-		Currency targetOperatingCurrency;
-		if ("fiat".Equals(currencyStr, StringComparison.OrdinalIgnoreCase))
```

Comment:
`convert-to` makes `currency` field not important since it changes the mode invoice issuing operates. @NicolasDorier brought up a point that FIAT can be confusing to users - easier just to directly set desired currency and not rely on autodetection

Reply:
In this context, `convert-to` is the same as the `currency` field. 
My point is that users e.g. outside of US see dollar $ sign everywhere (in the Strike app and dashboard). 
So they will try to configure `USD` here, and it will start failing much later on API calls because, in reality, they have a `USDT` balance. 

I plan to add some nice combobox with available currencies under Strike configuration page, but need to first push internally to expose some new Strike API.